backend/src/test/services/AssignedService.test.ts

```diff
@@ -0,0 +1,73 @@
+import 'reflect-metadata';
+import { mock } from 'jest-mock-extended';
+import { sequelizeMock } from '../helpers/SequelizeMock';
+import AssignedRepository from '../../main/repositories/AssignedRepository';
+import { container } from 'tsyringe';
+import { MultipleAssignedCreationDTO } from '../../main/dto/AssignedDTOs';
+import { Assigned } from '../../main/models/Assigned';
+import { AssignedService } from '../../main/services/AssignedService';
+
+describe('AssignedService tests', () => {
+  let assignedRepositoryMock: any = null;
+  beforeAll(() => {
```

Comment:
Same here add space before functions. This comment applies for all the test files